src/Mentat.UI/Views/Mentor/AddAssignment.cshtml

```diff
@@ -1,34 +1,78 @@
-﻿@model Mentat.UI.Models.Assignment
+﻿@using Microsoft.EntityFrameworkCore.Infrastructure
+@using System.Reflection
+@model Mentat.UI.Models.Assignment
 
 @{
-    ViewData["Title"] = "Add Assignment";
+    ViewData["Title"] = "AddAssignment";
+    Layout = "~/Views/Shared/_Layout.cshtml";
 }
 
 @using (Html.BeginForm("SubmitForm", "Mentor", FormMethod.Post))
 {
     <h2>Add Assignment Form</h2>
+
+    <!-- ID FIELD -->
+    
+    @Html.DisplayName("Id")
+    <div>
+        @Html.TextBoxFor(model=>model.Id)
+        @Html.ValidationMessageFor(model => model.Id,
+        "", new { @class = "text-danger" })
+    </div>
+    
+    
+    <!-- MENTOR NAME FIELD -->
+    
+    @Html.DisplayName("MentorName")
+    <div>
+        @Html.TextBoxFor(model => model.MentorName)
+        @Html.ValidationMessageFor(model => model.MentorName,
+            "", new { @class = "text-danger" })
+    </div>
+
+    <!-- ASSIGNMENT NAME FIELD -->
+    
+    @Html.DisplayName("AssignmentName")
     <div>
-        Mentor Name
+        @Html.TextBoxFor(model => model.AssignmentName)
+        @Html.ValidationMessageFor(model => model.AssignmentName,
+        "", new { @class = "text-danger" })
     </div>
-    @Html.TextBoxFor(model => model.MentorName)
+
+    <!-- ASSIGNMENT FILE TYPE FIELD -->
+    @Html.DisplayName("AssignmentType")
     <div>
-        Assignment Name
+        @Html.RadioButtonFor(m => m.AssignmentType, "c")
+        <span>C</span>
     </div>
-    @Html.TextBoxFor(model => model.AssignmentName)
     <div>
-        Assignment Type
+        @Html.RadioButtonFor(m => m.AssignmentType, "c++")
+        <span>C++</span>
     </div>
-    @Html.RadioButtonFor(m => m.AssignmentType, ".c") @:.c
-    @Html.RadioButtonFor(m => m.AssignmentType, ".cpp") @:.cpp
+    
+
+    <!-- SELECT & UPLOAD SAMPLE FILES -->
+    @Html.DisplayName("SampleExecutableName")
     <div>
-        Sample Executable Name
+        @Html.TextBoxFor(model => model.SampleExecutableName)
+        @Html.ValidationMessageFor(model => model.SampleExecutableName,
+        "", new { @class = "text-danger" })
     </div>
-    @Html.TextBoxFor(model => model.SampleExecutableName)
+    
+    <!--SELECT & UPLOAD TEST FILES -->
+    @Html.DisplayName("TestFileNames")
     <div>
-        Test File Names
+        @Html.TextBoxFor(model => model.TestFileNames)
+        @Html.ValidationMessageFor(model => model.TestFileNames,
+        "", new { @class = "text-danger" })
     </div>
-    @Html.TextAreaFor(model => model.TestFileNames)
+
     <div>
-    <input type="submit" value="Submit" />
+        <input type="submit" value="Create" class="btn btn-primary" />
     </div>
+
+}
+
+@section Scripts {
+    @{await Html.RenderPartialAsync("_ValidationScriptsPartial");}
```

Comment:
I don't see a file being brought in called _ValidationScriptsPartial. Can you fill me in on this?

Reply:
Without this the validation would not show up as an overlay in the view. This is just a name autogenerated that calls some scripts form the library that already exist within the project.

Reply:
I see, the file _ValidationScriptsPartial is already in the project. Thanks